src/pages/ProductPage/ProductPage.tsx

```diff
@@ -49,37 +59,50 @@ export const ProductPage = () => {
         .then((data: Device[]) => {
           const foundProduct = data.find(item => item.id === slug);
 
-          setProduct(foundProduct || null);
+          setDevice(foundProduct || null);
         })
         .catch(error => {
           console.error('There was a problem with the fetch operation:', error);
         });
     }
   }, [category, slug]);
 
-  if (!product) {
+  if (!device) {
     return <div>Loading...</div>;
   }
 
   return (
-    <div>
+    <div className={styles.container}>
       <div className={styles.breadCrumbs}>
-        <BreadCrumbs productName={product.name} />
-      </div>
-      <div className="description">
-        <TemporaryProductPage />
+        <BreadCrumbs productName={device.name} />
       </div>
 
-      <div className={styles.description}>
-        <AboutSection description={product.description} />
-      </div>
-      <div className={styles.techSpecs}>
-        <TechSpecsSection product={product} />
+      <div className={styles.buttonBack}>
+        <BackButton />
       </div>
+
+      <h1 className={styles.title}>{device.name}</h1>
+
+      <section className={styles.sliderWrapper}>
+        <ProductPhotosSlider photos={device.images} productName={device.name} />
+      </section>
+
+      <section className={styles.paramsSelectionWrapper}>
+        <ParamsSelection device={device} cartProduct={selectedProduct!} />
+      </section>
+
+      <section className={styles.about}>
+        <AboutSection description={device.description} />
+      </section>
+
+      <section className={styles.techSpecs}>
+        <TechSpecsSection product={device} />
+      </section>
+
       <div className={styles.recommended}>
```

Comment:
If we have all these components inside sections, let's do this one the same way.
You've done it for other sections, but not this one, why?
The component itself inside could be wrapped in a react fragment instead.